gazelle/manifest/generate/generate.go

```diff
@@ -173,12 +167,7 @@ func writeOutput(
 	manifestGeneratorHashPath string,
 	requirementsPath string,
 ) error {
-	stat, err := os.Stat(outputPath)
-	if err != nil {
-		return fmt.Errorf("failed to write output: %w", err)
-	}
-
-	outputFile, err := os.OpenFile(outputPath, os.O_WRONLY|os.O_TRUNC, stat.Mode())
+	outputFile, err := os.OpenFile(outputPath, os.O_WRONLY|os.O_TRUNC|os.O_CREATE, 0666)
```

Comment:
Why 666 and not 644? Having a more restrictive permission may be nicer.

Reply:
I picked 0666 because apparently this is what the higher-level Golang APIs use when you don't specify a mask. It sounds like umask gets applied here so it will probably change to 0644 anyway. I can change to 0644 in the code though.